ethportal-peertest/src/jsonrpc.rs

```diff
@@ -152,6 +158,22 @@ impl JsonRpcEndpoint {
                     Value::Array(vec![json!(256)]),
                 ]),
             },
+            JsonRpcEndpoint {
+                method: "portal_historyStoreContent".to_string(),
+                id: 10,
+                params: Params::Array(vec![
+                    Value::String("000f00d1c390624d3bd4e409a61a858e5dcc5517729a9170d014a6c96530d64dd8621d".to_string()),
+                    Value::String("01".to_string()),
+                ]),
+            },
+            JsonRpcEndpoint {
+                method: "portal_stateStoreContent".to_string(),
+                id: 11,
+                params: Params::Array(vec![
+                    Value::String("0044000000b8be7903aee73b8f6a59cd44a1f52c62148e1f376c0dfa1f5f773a98666efc2bd1c390624d3bd4e409a61a858e5dcc5517729a9170d014a6c96530d64dd8621d01020001".to_string()),
+                    Value::String("02".to_string()),
+                ]),
+            },
```

Comment:
Agreed, it just gets pretty messy with this current setup. You pretty much end up with a nested monstrosity containing all the raw byte vectors needed to generate the content key. I've added this as a todo to be cleaned up post #295 for now if that works.

Reply:
I was thinking that it would be straightforward to define a few `const` for some block hash, address, etc. in the same way that we have `const` for data radius.

Reply:
Gotcha, the only side-effect is that I had to label the `AccountTrieNode` struct fields as `pub`, but that doesn't seem like a problem imo.